pkg/operation/botanist/monitoring.go

```diff
@@ -203,10 +222,31 @@ func (b *Botanist) DeploySeedMonitoring(ctx context.Context) error {
 				}
 			}
 		}
+
+		var (
+			defaultSecretName  = "alertmanager-tls"
```

Comment:
Maybe expose this as a constant somewhere and reuse it here and in `pkg/operation/botanist/secrets.go` where it is generated?